command/cp.go

```diff
@@ -134,6 +134,14 @@ var copyCommandFlags = []cli.Flag{
 		Name:  "acl",
 		Usage: "set acl for target: defines granted accesses and their types on different accounts/groups",
 	},
+	&cli.StringFlag{
```

Comment:
Could you please add a sample usage for both `cache-control` and `expires` to the help text?